ProjFS.Mac/PrjFSKextTests/KextMockUtilities.hpp

```diff
@@ -3,6 +3,14 @@
 #include <unordered_map>
 #include <unordered_set>
 
+namespace KextMock
+{
+    struct PlaceholderValue
+    {};
+    
+    extern PlaceholderValue _;
```

Comment:
Sure, that commit was thrown together as quickly as possible so that @jeschu1 had something to work with, I think there are a few things there that could be improved. I'll try and get that done soon.

Reply:
If you get pressed for time @pmj I can do it

Reply:
Comment added :-)

Reply:
Apologies for bringing this up again, but could you add a few more details (and possibly rename `PlaceholderValue`)?  Someone new to the code might think that `PlaceholderValue` is related to file placeholders based on its name.

Maybe something like:

```
namespace KextMock
{
    struct ParameterPlaceholderValue
    {};

    // Matches with any parameter value
    // ParameterPlaceholderValue can be used:
    //   - When calling kext functions, for parameters values that do not matter to the test
    //   - When checking if a function was called (i.e. DidCallFunction).  ParameterPlaceholderValue is 
    //     a match for all parameter values.
    extern ParameterPlaceholderValue _;
```

Reply:
Updating with suggested comment